gen/o365

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/perl
+
+# Generates members of o365 service in JSON format
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use JSON::XS;
+
+sub processMembers;
+
+our $SERVICE_NAME     = "o365";
+our $PROTOCOL_VERSION = "3.0.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data      = perunServicesInit::getHierarchicalData;
+
+our $A_USER_FIRST_NAME;         *A_USER_FIRST_NAME =            \'urn:perun:user:attribute-def:core:firstName';
+our $A_USER_LAST_NAME;          *A_USER_LAST_NAME =             \'urn:perun:user:attribute-def:core:lastName';
+our $A_USER_DISPLAY_NAME;       *A_USER_DISPLAY_NAME =          \'urn:perun:user:attribute-def:core:displayName';
+our $A_USER_PREFERRED_MAIL;     *A_USER_PREFERRED_MAIL =        \'urn:perun:user:attribute-def:def:preferredMail';
+our $A_USER_USER_TYPE;          *A_USER_USER_TYPE =             \'urn:perun:user:attribute-def:def:o365-userType';
+our $A_USER_ASSIGNED_LICENSES;  *A_USER_ASSIGNED_LICENSES =     \'urn:perun:user:attribute-def:def:o365-assignedLicenses';
```

Comment:
@licehammer: could you explain me please, why it shouldn't it be user's attribute? I thought every member of a group can have various assigned licenses